### PR TITLE
Allow updating an existing k8s secret in addition to creating a new one.

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

File: pkg/cosign/kubernetes/secret.go [hunk 1]
```diff
@@ -22,6 +22,8 @@ import (
 	"strings"
 
 	kubernetesclient "github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes/client"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+
```
Comment (cpanato) [inline pkg/cosign/kubernetes/secret.go:26]: very nit, maybe we can organize the import a bit

```go
import (
	"context"
	"fmt"
	"io/ioutil"
	"os"
	"strings"

	"github.com/pkg/errors"
        v1 "k8s.io/api/core/v1"
	k8serrors "k8s.io/apimachinery/pkg/api/errors"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
        kubernetesclient "github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubernetes/client"

	"github.com/sigstore/cosign/pkg/cosign"
)
```

**Review by cpanato — APPROVED** [review]
missing the DCO

File: pkg/cosign/kubernetes/secret.go (hunk 1 above)
Comment (dlorenc) [reply pkg/cosign/kubernetes/secret.go]: Thanks!

**Review by priyawadhwa — APPROVED** [review]
lgtm!

**Pull request merged**